iroh/src/endpoint.rs

```diff
@@ -1060,6 +1057,18 @@ impl Endpoint {
         &self.msock.metrics
     }
 
+    /// Returns addressing information about a recently used remote endpoint.
```

Comment:
maybe mention that this is a snapshot in time, or sth, to indicate that this can have already changed by the moment it is used